### PR TITLE
Add raw parameter to S3Download

### PR DESCRIPTION
**Pull request by tonal**
## Summary
see https://github.com/PrefectHQ/prefect/discussions/4226

## Checklist

This PR:

- [ ] adds new tests (if appropriate)
- [ ] adds a change file in the `changes/` directory (if appropriate)
- [ ] updates docstrings for any new functions or function arguments, including `docs/outline.toml` for API reference docs (if appropriate)

### CODE REVIEW

File: src/prefect/tasks/aws/s3.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def run(
         credentials: str = None,
         bucket: str = None,
         compression: str = None,
+        raw: bool = False,
```
Comment (madkinsz) [inline src/prefect/tasks/aws/s3.py:46]: I think I would prefer `as_bytes` but it may be worth waiting for @jcrist to weigh in.

**Review by madkinsz — COMMENTED** [review]
Thanks for contributing! This will also need a test and a changes/ entry.

File: src/prefect/tasks/aws/s3.py (hunk 1 above)
Comment (jcrist) [reply src/prefect/tasks/aws/s3.py:46]: Yeah, `as_bytes` would be preferred here. This probably should have been the default originally, but that ship has sailed.

**Review by jcrist — COMMENTED** [review]
Can you also add a changelog entry? See https://docs.prefect.io/core/development/contributing.html#changelog.

**Review by jcrist — APPROVED** [review]
I pushed the rename to `as_bytes` and added a changelog entry. Thanks for the PR!

**Pull request merged**